SU-HSDO/suhumsci
# SHS 5119 jQuery UI dialog z-index

# READY FOR REVIEW

## Summary
This PR updates the jquery ui variables inside the custom theme. The issue seems to be related to 
https://www.drupal.org/project/gin_lb/issues/3351882

## Urgency
medium

## Steps to Test
1.Visit a Layout Builder Page like https://hs-colorful.suhumsci.loc/admin/structure/types/manage/hs_person/display/default/layout
2. Click on a  Pencil icon next to one of the blocks, and then click "Configure"
3. Ensure that the modal window appears in front of the grayed-out panel (instead of behind it.) 

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
